pkgs/development/ocaml-modules/junit/shared.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, fetchFromGitHub }:
+rec {
+  version = "2.0.2";
+
+  src = fetchFromGitHub {
+    owner = "Khady";
+    repo = "ocaml-junit";
+    rev = version;
+    sha256 = "sha256-QbcHGjZGlcGfzGdINLOCNcGZCxxqjVpouAnmrP4/VbE=";
+  };
+
+  meta = with lib; {
+    description = "ocaml-junit is an OCaml package for the creation of JUnit XML reports, proving a typed API to produce valid reports acceptable to Jenkins, comes with packages supporting OUnit and Alcotest.";
+    license = licenses.gpl2;
```

Comment:
Use one of the clearer updates `gpl2Only` or `gpl2Plus` (“GPL 2 or any later version”).

Reply:
https://github.com/Khady/ocaml-junit/blob/master/LICENSE

Which sort of license is this?

Reply:
`lib.licenses.lgpl3Plus` according to opam.